internal/app/mapper/mapper_test.go

```diff
@@ -65,6 +70,96 @@ var postivetests = []TableEntry{
 			stringFragment,
 		},
 	},
+	{
+		Description: "RequestNodeMatch with node id exact match",
+		Parameters: []interface{}{
+			getRequestNodeExactMatch(nodeIDField, nodeid),
+			getResultStringFragment(),
+			clusterTypeURL,
+			stringFragment,
+		},
+	},
+	{
+		Description: "RequestNodeMatch with node cluster exact match",
+		Parameters: []interface{}{
+			getRequestNodeExactMatch(nodeClusterField, nodecluster),
+			getResultStringFragment(),
+			clusterTypeURL,
+			stringFragment,
+		},
+	},
+	{
+		Description: "RequestNodeMatch with node region exact match",
+		Parameters: []interface{}{
+			getRequestNodeExactMatch(nodeRegionField, noderegion),
+			getResultStringFragment(),
+			clusterTypeURL,
+			stringFragment,
+		},
+	},
+	{
+		Description: "RequestNodeMatch with node zone exact match",
+		Parameters: []interface{}{
+			getRequestNodeExactMatch(nodeZoneField, nodezone),
+			getResultStringFragment(),
+			clusterTypeURL,
+			stringFragment,
+		},
+	},
+	{
+		Description: "RequestNodeMatch with node subzone exact match",
+		Parameters: []interface{}{
+			getRequestNodeExactMatch(nodeSubZoneField, nodesubzone),
+			getResultStringFragment(),
+			clusterTypeURL,
+			stringFragment,
+		},
+	},
+	{
+		Description: "RequestNodeMatch with node id regex match",
+		Parameters: []interface{}{
+			getRequestNodeRegexMatch(nodeIDField, "n....d"),
+			getResultStringFragment(),
+			clusterTypeURL,
+			stringFragment,
+		},
+	},
+	{
+		Description: "RequestNodeMatch with node cluster regex match",
+		Parameters: []interface{}{
+			getRequestNodeRegexMatch(nodeClusterField, "c.....r"),
```

Comment:
Can we try some other regex variations in a few of these tests? ex: `c*`

Reply:
sounds good. Added some more.